cmd/test-cli/main.go

```diff
@@ -114,19 +119,19 @@ func doGetPayload(v validatorPrivateData, boostEndpoint string, beaconNode Beaco
 		log.WithError(err).Fatal("could not sign blinded beacon block")
 	}
 
-	payload := boostTypes.SignedBlindedBeaconBlock{
+	payload := consensusapi.SignedBlindedBeaconBlock{
 		Message:   &blindedBeaconBlock,
-		Signature: signature,
+		Signature: phase0.BLSSignature(signature),
 	}
-	var respPayload boostTypes.GetPayloadResponse
+	var respPayload api.VersionedExecutionPayload
 	if _, err := server.SendHTTPRequest(context.TODO(), *http.DefaultClient, http.MethodPost, boostEndpoint+"/eth/v1/builder/blinded_blocks", "test-cli", payload, &respPayload); err != nil {
 		log.WithError(err).Fatal("could not get payload")
 	}
 
-	if respPayload.Data == nil {
+	if respPayload.Bellatrix == nil {
 		log.Fatal("Did not receive correct payload")
 	}
```

Comment:
Not related to the changes in your PR and not a major security issue, but if the `respPayload.Bellatrix` is nil there will be a segfault at line 134.
The following suggestion fixes the issue.

```suggestion
		log.Fatal("Did not receive correct payload")
		return
	}
```